manifests/m/Microsoft/SurfacePro10forBusinessUpdate24/083/33185/0(64bit)/24.083.33185.0/Microsoft.SurfacePro10forBusinessUpdate24.083.33185.0(64bit).locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.SurfacePro10forBusinessUpdate24.083.33185.0(64bit)

```

```suggestion
PackageIdentifier: Microsoft.SurfacePro10forBusinessUpdate
```
Please update file name and path to match this change.